documents/getting-started-ubuntu.md

```diff
@@ -0,0 +1,99 @@
+# Getting Started: Ubuntu
+
+## Prerequisites
+#### Common: Installing the NNStreamer Development Kit package via APT
+The NNStreamer packages for Ubuntu are released at a [PPA](https://launchpad.net/~nnstreamer/+archive/ubuntu/ppa) repository. In order to add this APT repository to your system,
+
+```bash
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt update
+$ sudo apt install nnstreamer-dev
+```
+
+- [NNStreamer's PPA](https://launchpad.net/~nnstreamer/+archive/ubuntu/ppa) does not provide the packages for Ubutnu 20.04.
+
+#### Installing the base packages for ROS2
+
+Currently, the Ubuntu packages for Eloquent Elusor are provided only for 18.04 by [the official APT repository](http://packages.ros.org/ros2/ubuntu).
+
+```bash
+$ sudo apt update && sudo apt install curl gnupg2 lsb-release
+$ curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+$ sudo sh -c 'echo "deb [arch=$(dpkg --print-architecture)] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" > /etc/apt/sources.list.d/ros2-latest.list'
+$ sudo apt update
+```
+
+Then, you can install the ROS2 Desktop or Base virtual packages as follows:
+```bash
+$ sudo apt install ros-eloquent-ros-base
+```
+or
+```bash
+$ sudo apt install ros-eloquent-ros-desktop
+```
+
+#### Installing the base packages for ROS
+The ROS distros namely Kinetic Kame, Melodic Morenia, and Noetic Ninjemys are supported by Ubuntu 16.04, 18.04, and 20.04, respectively via [the official APT repository](http://packages.ros.org/ros/ubuntu) for ROS.
+
+```bash
+$ sudo apt update && sudo apt install curl gnupg2 lsb-release
+$ wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+$ sudo sh -c 'echo "deb [arch=$(dpkg --print-architecture)] http://packages.ros.org/ros/ubuntu $(lsb_release -cs) main" > /etc/apt/sources.list.d/ros-latest.list'
+$ sudo apt update
+```
+
+Then, you can install the ROS Desktop or Base virtual packages as follows:
+
+```bash
+$ sudo apt install ros-${ROSDIST}-ros-base
+```
+or
+```bash
+$ sudo apt install ros-${ROSDIST}-ros-desktop
+```
+
+Where the ```ROSDIST``` is an environment variable indicating the code name of the ROS distros, kinetic (16.04), melodic (18.04), and noetic (20.04).
+
+
+## Installing binary packages using APT
+The Ubuntu pakcages for ```nnstreamer-ros2``` and ```nnstreamer-ros``` are released at [this repository](https://bintray.com/beta/#/nnsuite/nnstreamer-ros?tab=packages) powered by JFrog Bintray. To add this repository into your system,
+```bash
+$ echo "deb [trusted=yes] https://dl.bintray.com/nnsuite/nnstreamer-ros  $(lsb_release -cs) main" | sudo tee -a /etc/apt/sources.list.d/nns-ros.list
```

Comment:
### Comment 01: 
As we all know, the Tizen repo does not have a GPG key in its official repository. BTW, Do we also append the "**[trusted=yes]**" in the official dl.bintray.com?


### Comment 02: 
* https://dl.bintray.com/nnsuite/nnstreamer-ros/dists/
  * bionic
  * xenail <---- HERE.!!! It's weird. @wooksong , Please check it.
  * xenial 

* Screenshot:
![image](https://user-images.githubusercontent.com/82404/99346959-32c94e80-28d9-11eb-9bdc-6e3cbc2ad447.png)

Reply:
@leemgs 
For Comment 01, For Comment 01, if you are using Ubuntu 16.04, I guess it would be OK without ```[trusted=yes]```. But ```apt update```  would be failed without ```[trusted=yes]``` since Ubuntu has changed the policy more tightly after 18.04.

For Comment 02, I made it by mistake. Now it has been removed:)